<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskView</class>
 <widget class="QWidget" name="TaskView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QLabel" name="taskResultLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <italic>false</italic>
       <bold>true</bold>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 18pt &quot;Segoe UI&quot;; color: green</string>
     </property>
     <property name="text">
      <string>Верно</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTextEdit" name="textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QLabel" name="whoseAnswersLabel">
     <property name="text">
      <string>Ответ: мой</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>605</width>
        <height>207</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="priceLabel">
     <property name="text">
      <string>Стоимость:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>115</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="switchAnswersButton">
     <property name="text">
      <string>Показать правильный ответ</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>114</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="taskNumlabel">
     <property name="text">
      <string>Задание:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
